CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - **CUMULUS-1212**
   - `@cumulus/post-to-cmr` will now fail if any granules being processed are missing a metadata file. You can set the new config option `skipMetaCheck` to `true` to pass post-to-cmr without a metadata file.
 
+- **CUMULUS-1193**
+  - The elasticsearch instance is moved behind the VPC. There are two migration options, both will incur loss of data:
+    - Create a completely new stack
+    - Deploy to your current stack. This will create a new ES instance and remove your current instance and WILL RESULT IN LOSS OF DATA. You will need to manually remove data from the following DynamoDb tables: Providers, Collections, Granules, Files, Pdrs, Rules, and Executions to stay in sync with ES and regenerate all data.
```

Comment:
I'm thinking I should remove option #2 for end-users, since there can be some weirdness if you don't just create the new stack.

Reply:
Given the conversation today in the architecture meeting and my changes that will require deleting stacks, it seems like we should just say that deleting your stack is required. Should this be flagged under a `** BREAKING CHANGES **` header or something?

Reply:
It is currently under the `BREAKING CHANGES` header.